commit/merkleroot/rmn/client.go

```diff
@@ -104,19 +105,21 @@ func (c *client) ComputeReportSignatures(
 			data:     updateReq,
 			rmnNodes: mapset.NewSet[NodeID](),
 		}
-		for _, node := range c.rmnCfg.Home.RmnNodes {
+		for _, node := range c.rmnCfg.Home.Nodes {
 			if node.SupportedSourceChains.Contains(cciptypes.ChainSelector(updateReq.LaneSource.SourceChainSelector)) {
 				updatesPerChain[updateReq.LaneSource.SourceChainSelector].rmnNodes.Add(node.ID)
 			}
 		}
-	}
 
-	// Filter out the lane update requests for chains without enough RMN nodes supporting them.
-	for chain, l := range updatesPerChain {
-		if l.rmnNodes.Cardinality() < c.rmnCfg.Remote.MinObservers {
-			c.lggr.Warnw("not enough RMN nodes for chain",
-				"chain", chain, "minObservers", c.rmnCfg.Remote.MinObservers)
-			delete(updatesPerChain, chain)
+		// Filter out the lane update requests for chains without enough RMN nodes supporting them.
+		for chain, l := range updatesPerChain {
+			if l.rmnNodes.Cardinality() <
+				int(c.rmnCfg.Home.MinObservers[cciptypes.ChainSelector(updateReq.LaneSource.SourceChainSelector)]) {
```

Comment:
I see similar access repeated multiple times with different variables. Maybe add a method to `cfg.Home` that takes a uint64 and returns the int instead.